src/pages/ReportDetailsPage.js

```diff
@@ -108,7 +108,7 @@ function ReportDetailsPage(props) {
                     }
                 },
             });
-        } else if ((!participants.length || !isPolicyMember) && isUserCreatedPolicyRoom && !props.report.parentReportID) {
+        } else if (isUserCreatedPolicyRoom && (!participants.length || !isPolicyMember) && !props.report.parentReportID) {
```

Comment:
Why is the order changed here?

Reply:
Because sometimes the `participants` won't exist if the report is a User Created Policy Room